### PR TITLE
MyGet: Refactors HybridRow dependency to use Nuget.org

### PR DESCRIPTION
**Pull request by j82w**
# Pull Request Template

## Description

Removing the myget dependency on hybrid row nuget and consuming it via nuget.org.

## Type of change

Please delete options that are not relevant.

- [] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] This change requires a documentation update

## Closing issues

To automatically close an issue: closes #IssueNumber

### CODE REVIEW

File: NuGet.config [hunk 1]
```diff
@@ -2,6 +2,5 @@
 <configuration>
   <packageSources>
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
```
Comment (ealsur) [inline NuGet.config:4]: Is this file needed at all now?

**Review by ealsur — APPROVED** [review]
Awesome!

File: NuGet.config (hunk 1 above)
Comment (j82w) [reply NuGet.config:4]: I'm leaving it because it will ensure that the project always uses the correct nuget source instead of depending on the local machines configuraiton.
